Gaijinizer patch/CommonEvents_63.txt

```diff
@@ -4,59 +4,61 @@
 
 はげしっ…！！すぎぃ…ぃ！！♡
 > CONTEXT: CommonEvents_63/4/Dialogue/kazuya-0
-Don't wait!!Moth. Haha ...!!Too much...ぃ! !
-Moth#MTLed
+Uuugh!! ♡ Aaah!! ♡ I can't...!! ♡
+Too... rough...!! ♡ #Ron
 > END STRING
 
 > BEGIN STRING
 ええ、そうですよ…♡
 
 あなたを始末すること…、それが私の役目ですから♡
 > CONTEXT: CommonEvents_63/8/Dialogue/yuuma
-Yes, yes...そ う. Killing you...that is my role#MTLed
+Yes, that's right... ♡
+
+My role is to dispose of you. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 このまま一気に搾り取って…
 
 あなたを快楽で狂わせながら、私の胸で眠らせてあげます♡
 > CONTEXT: CommonEvents_63/12/Dialogue/yuuma
-Just squeeze all at once...I will make you
-sleep in my heart while making you go crazy with
-pleasure#MTLed
+I'll wring you out like this in one go...
+I'll put you to sleep with my chest while I make
+you go mad with pleasure. ♡#Ron
 > END STRING
```

Comment:
I'm not the best of translators, but some translations I've seen of this mention "in her chest" as opposed to "with". It makes sense in context as well, because Iron Maiden is keeping Kazuya inside of her....

I'd suggest trying to revise the translation a bit, then I'll check it again, if I'm understanding this correctly.

( "Put to sleep" can certainly be meant as euphemism which can have a deadly or sinister tone as well, but I'd like to see that you throw at me first.)